ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -25,9 +25,6 @@ images:
 - dockerfile_path: images/cli/Dockerfile.rhel
   from: base
   inputs:
-    bin:
```

Comment:
@sallyom you need a followup to https://github.com/openshift/oc/pull/218/files#diff-664dbcc5153972447bb795b2ee4d1163R1

Reply:
That was fixed in https://github.com/openshift/release/pull/9756

Reply:
it missed master I think

Reply:
Yes it did, fixed in https://github.com/openshift/release/pull/10224